src/Modules/CalcBreakdown.lua

```diff
@@ -192,7 +192,7 @@ function breakdown.leech(instant, instantRate, instances, pool, rate, max, dur,
 	if actor.mainSkill.skillData.showAverage then
 		if instant > 0 then
 			if instantLeechProportion ~= 1 then 
-				t_insert(out, s_format("Instant Leech: %.1f ^8(%d%% x %.1f)", instant, instantLeechProportion * 100, dur * pool * data.misc.LeechRateBase))
+				t_insert(out, s_format("Instant Leech: %.1f ^8(%d%% x %.1f)", instant, instantLeechProportion * 100, dur * pool * data.misc.LeechRateBase / (1-instantLeechProportion)))
```

Comment:
```suggestion
				t_insert(out, s_format("Instant Leech: %.1f ^8(%d%% x %.1f)", instant, 100, dur * pool * data.misc.LeechRateBase / (1/instantLeechProportion - 1)))
```

Reply:
Thanks!

Your proposed change seems to recalculate the `instant`-value, which would make the breakdown look like
`instant leech (100% * instant leech)`.

![image](https://user-images.githubusercontent.com/48551928/231487869-74734069-dd5a-4dc2-aa7d-e41c69281402.png)

I think it makes more sense to show `instant leech (instant leech % * size of leech instance)`.

Reply:
Apologies I missed that comma yes you are correct.